JKH/Day2/가운데 글자 가져오기.js

```diff
@@ -0,0 +1,12 @@
+function solution(s) {
+  var answer = '';
+    
+  if (s.length % 2 === 0) {
+    answer += s[s.length / 2 - 1];
+    answer += s[s.length / 2];
+  } else {
+    answer += s[Math.floor(s.length / 2)];
+  }
+    
+  return answer;
+}
```

Comment:
```suggestion
function solution(s) {
  let answer = '';
  const wordLength = s.length;
    
  if (wordLength % 2 === 0) {
    answer += s[wordLength / 2 - 1];
    answer += s[wordLength / 2];
  } else {
    answer += s[Math.floor(wordLength / 2)];
  }
    
  return answer;
}
```

전체적으로 `s.length`가 반복적으로 사용됩니다. 변수로 빼서 사용하면 다음과 같은 이점이 있습니다.
- 해당 변수의 의미가 명확해짐
- 가독성 향상
- 유지보수 향상

사실 이부분은 변수로 빼줄 필요는 없지만, 이러한 사소한 습관이 나중에 좋은 코드 습관으로 연결된다고 생각합니다😊

Reply:
작성할 때 사실 도트 연산자가 너무 많아서 나중에 볼 때 가독성이 좋지 않겠다고 생각했었는데, 변수로 빼서 사용하는 편이 더 의미가 확실하고 빠르게 전달되는 것 같습니다. 앞으로 응용하도록 하겠습니다. 리뷰 감사합니다!😄